src/main/resources/static/js/src/HistoryController.js

```diff
@@ -0,0 +1,36 @@
+const app = angular.module("myApp", []);
+app.controller("HistoryController", function($scope,$http) {
+    let myPhone = document.getElementById("from").textContent;
+
+    $http({
+        method: 'GET',
+        url: '/rest/getCallsList'+myPhone
```

Comment:
Don't do so.
How will:  /rest/getCallsList+324234552342??? 
Really? :)) 
Every number will have API? 
because if you will have: 
/rest/getCallsList+324234552342 - first router
/rest/getCallsList+8435242342d435 - second router.

Read: https://spring.io/guides/tutorials/rest/